.nvmrc

```diff
@@ -1 +1 @@
-lts/*
+20
```

Comment:
This makes sure the version corresponds to the one Vercel provides (as they are always late with LTS versions).